generators/app/templates/config/env.js

```diff
@@ -1,17 +1,27 @@
 'use strict';
 
+const _ = require('lodash');
+
 function getEnv() {
-  if (process.env.NODE_ENV) {
-    return process.env.NODE_ENV;
+  const NODE_ENV = process.env.NODE_ENV;
+  const AWS_LAMBDA_FUNCTION_NAME = process.env.AWS_LAMBDA_FUNCTION_NAME;
+
+  // Environment base on lambda function name
+  // TODO put your own lambda function name here
+
+  const AWS_LAMBDA_PRODUCTION_ENV = ['LAMBDA_FUNCTION_NAME_PROD']; // production
+  const AWS_LAMBDA_STAGING_ENV = ['LAMBDA_FUNCTION_NAME_STAGE'];  // staging
```

Comment:
this is not exactly the same behavior as before, we used to just set the environment to staging if there was any `AWS_LAMBDA_FUNCTION_NAME`, this now requires to configure an `LAMBDA_FUNCTION_NAME_STAGE`